src/modules/sync-cli.js

```diff
@@ -175,13 +176,13 @@ var self = {
     });
   },
 
-  setOptions: function(options) {
+  setOptions: function(config, options) {
     // Make sure config is initialised
-    if( ! self.config ) {
-      self.config = JSON.parse(JSON.stringify(self.defaults));
+    if( ! config ) {
+      config = JSON.parse(JSON.stringify(self.defaults));
```

Comment:
Can you check that `self.config` (the global config) is being set in the init call. It may not being set when changing this code.